cmd/containerd/main_unix.go

```diff
@@ -23,22 +23,35 @@ var handledSignals = []os.Signal{
 	unix.SIGPIPE,
 }
 
-func handleSignals(ctx context.Context, signals chan os.Signal, server *server.Server) error {
-	for s := range signals {
-		log.G(ctx).WithField("signal", s).Debug("received signal")
-		switch s {
-		case unix.SIGCHLD:
-			if err := reaper.Reap(); err != nil {
-				log.G(ctx).WithError(err).Error("reap containerd processes")
+func handleSignals(ctx context.Context, signals chan os.Signal, serverC chan *server.Server) chan struct{} {
```

Comment:
Uh, how did we get this duplicated between linux and unix? They're identical (linux should actually ignore `SIGPIPE` too)

Reply:
fixed